content/docs/concepts/low-power.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Low Power"
+linkTitle: "Low Power"
+type: "docs"
+weight: 35
+description: >
+  How to consume less power on microcontrollers
+---
+
+Microcontrollers are usually rather non-demanding when it comes to power consumption.
+Couple of watts is the most you may expect a hobby project would use.
+
+Sometimes, even that is not low enough.
+Power your project from a battery, and it'd benefit greatly from consuming less.
+
+Common scenario for such projects is sleeping most of the time, and only rarely briefly wake up to query sensors and execute actions.
+The main caveat here is if not configured properly, microcontroller and peripherals continue to consume power as usual while your program "sleeps".
+
+The standard way to pause execution in Go is to use `time.Sleep()` function that is available in TinyGo too. Depending on the microcontroller architecture and effort put into it, the efficiency of this function can vary. The only thing certain: this is better than busy looping until a certain time has passed.  
```

Comment:
> The only thing certain: this is better than busy looping until a certain time has passed.

I'm afraid not even that is true. Sometimes, `time.Sleep` is implemented as a busy loop until a given time has passed as a stopgap until proper sleeping has been implemented. But it's unlikely to be _worse_ at least.

Reply:
I'll try to re-formulate, thanks!